### DISCUSSION TITLE
getting started page not available

### DISCUSSION THREAD

**Discussion by Kubrix** [MonoGame/MonoGame#8273] [General]
https://monogame.net/
Press getting started - https://monogame.net/articles/getting_started/index.html
and get 404 **File not found**

**Comment by dellis1972** [comment]
@AristurtleDev @SimonDarksideJ is this fixed? 

Its now at https://docs.monogame.net/articles/getting_started/index.html

**Reply by SimonDarksideJ** [reply]
Would appear to be, while in its new home.  Tested link from the home page and no issues.
